### PR TITLE
fix issue in alignment task where selected scans is null

### PR DESCRIPTION
**Pull request by tnguy271**

### CODE REVIEW

**Comment by tnguy271** [comment]
@SteffenHeu Could you check this merge request?

File: src/main/java/io/github/mzmine/modules/dataprocessing/align_adap3/ADAP3AlignerTask.java [hunk 1]
```diff
@@ -240,6 +240,10 @@ private FeatureList alignPeaks() {
 
     alignedPeakList.getAppliedMethods().add(new SimpleFeatureListAppliedMethod(
         ADAP3AlignerModule.class, parameters, getModuleCallDate()));
+
+
+    allDataFiles.forEach(file-> alignedPeakList.setSelectedScans(file, file.getScans()));
```
Comment (SteffenHeu) [inline src/main/java/io/github/mzmine/modules/dataprocessing/align_adap3/ADAP3AlignerTask.java:245]: this is the same issue we discussed before, don't use all scans. Use the selected scans from the previous feautre lists used as input for this feature list.

Comment (tnguy271) [reply src/main/java/io/github/mzmine/modules/dataprocessing/align_adap3/ADAP3AlignerTask.java]: @SteffenHeu I changed the logic, please let me know if it's correct. Thank you.

Comment (SteffenHeu) [reply src/main/java/io/github/mzmine/modules/dataprocessing/align_adap3/ADAP3AlignerTask.java]: looks good, thanks

**Pull request merged**